docs/user/commandline_tools.rst

```diff
@@ -111,6 +112,20 @@ Get the specified feature type as an encoded feature schema string::
 
     $ geomesa getsft -u username -p password -c test_catalog -f test_feature --format spec
 
+keywords
+~~~~~~~~
+
+Add or remove keywords to a specified schema::
+Repeat the -a or -r flags to add or remove multiple keywords
+The ``--removeAll``` option removes all keywords
+The ```-l``` option lists the schema's keywords following all operations
+If there is whitespace within a keyword, enclose it in quotes for proper functionality
```

Comment:
The double colon needs to be at the end of this line for Sphinx to properly render the code block below.